src/Apps/Conversation/routes.tsx

```diff
@@ -0,0 +1,46 @@
+import loadable from "@loadable/component"
+import { graphql } from "react-relay"
+
+// @ts-ignore
+import { RouteConfig } from "found"
+
+export const conversationRoutes: RouteConfig[] = [
+  {
+    path: "/user/conversations",
+    getComponent: () => loadable(() => import("./ConversationApp")),
+    query: graphql`
+      query routes_ConversationQuery {
+        me {
+          ...ConversationApp_me
+        }
+      }
+    `,
+    prepareVariables: (params, props) => {
+      return {
+        first: 10,
+      }
+    },
+    cacheConfig: {
+      force: true,
+    },
+  },
+  {
+    path: "/user/conversations/:conversationID",
+    getComponent: () => loadable(() => import("./Routes/Detail")),
```

Comment:
#minor rather than `Detail` (which sort of adds another vocabulary word), what about a `Conversation` view/route?

Reply:
we already have a `conversation` component too i wonder if that makes it confusing, for now leaving this as is if its ok.